<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AICenter_UI_1Class</class>
 <widget class="QMainWindow" name="AICenter_UI_1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AICenter_UI_1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="leftMargin">
     <number>7</number>
    </property>
    <property name="topMargin">
     <number>7</number>
    </property>
    <property name="rightMargin">
     <number>7</number>
    </property>
    <property name="bottomMargin">
     <number>7</number>
    </property>
    <item>
     <widget class="QWidget" name="MainWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="TopWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="TitleBar" name="TitleWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="Device_Support_Button">
             <property name="text">
              <string>设备支持</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="AI_Assist_Button">
             <property name="text">
              <string>AI辅助</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Scene_Assist_Button">
             <property name="text">
              <string>场景辅助</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="Device_Support" name="page"/>
           <widget class="AI_Assist" name="page_2"/>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header>TitleBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AI_Assist</class>
   <extends>QWidget</extends>
   <header>AI_Assist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Device_Support</class>
   <extends>QWidget</extends>
   <header>Device_Support.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="AICenter_UI_1.qrc"/>
 </resources>
 <connections/>
</ui>
